server.js

```diff
@@ -9,13 +11,22 @@ const dbConnection = require('./features/dbConnection');
 
 const server = express();
 
+global.appRoot = path.resolve(__dirname);
 server.set('port', port);
 
 server.use(express.static('public'));
 
 // parse application/json
 server.use(express.json());
 
+server.use(fileUpload({
+	useTempFiles : true,
+	tempFileDir : '/tmp/'
+}));
+
+server.use(bodyParser.json()); // for parsing application/json
+server.use(bodyParser.urlencoded({ extended: true })); // for parsing application/x-www-form-urlencoded
```

Comment:
Should be included directly in express, pls double check
```suggestion
server.use(express.urlencoded({ extended: true })); // for parsing application/x-www-form-urlencoded
```